docs/body.md

```diff
@@ -252,7 +252,7 @@ Choose that option and you'll be presented with a dialog asking for the
 following inputs.
 
 * Device To Create - non-empty, non-ip, non-dns, unique name to use for this device in Zenoss. ''See note below''.
-* Auth URL - A keystone URL, such as http://\<hostname\>:5000/v2.0/ for Keystone v2, or http://\<hostname\>:5000/v3/ for Keystone v3.
+* Auth URL - A keystone URL, such as http://\<hostname\>:5000/ (latest supported API version will be selected by default, to force an API version, add the path to the end of the URL, like http://\<hostname\>:5000/v3/ or http://\<hostname\>:5000/v2.0/).
```

Comment:
OK, so this is debatable, but my feeling on this is that the default behavior should be for them to specify the auth url that is the proper auth url to use.  This will be the one that they are using in the CLI, the OS_AUTH_URL in the openrc that horizon spits out, etc..  Autodiscovery based on available versions is a fallback that we do just in case the users didn't give us the right URL, not a default behavior.